lua/entities/acf_gearbox/init.lua

```diff
@@ -873,12 +873,22 @@ do -- Braking ------------------------------------------
 		local Phys = Wheel:GetPhysicsObject()
 
 		if not Phys:IsMotionEnabled() then return end -- skipping entirely if its frozen
+		if not Link.LastVel then Link.LastVel = 0 end
```

Comment:
Why not put these two inside the link table when it gets created?

Reply:
I didn't look far enough to find it, but yes, that could certainly be done